sdk/examples/how_tos/advanced_transactions/advanced_transaction.rs

```diff
@@ -24,11 +25,13 @@ async fn main() -> Result<()> {
     // This example uses secrets in environment variables for simplicity which should not be done in production.
     dotenvy::dotenv().ok();
 
-    // Create the wallet
-    let wallet = Wallet::builder().finish().await?;
+    // Get the wallet we generated with `create_wallet`.
```

Comment:
No create_wallet and get anymore :trollface:

Reply:
doesn't a wallet have to be created still?

Reply:
Well the wallet still gets build again, we don't get it from somewhere